tests/unit/BootstrapMFAProviderTest.php

```diff
@@ -32,17 +32,39 @@ public function testUpdateTokensWithoutMember()
         $provider = Injector::inst()->get(BootstrapMFAProvider::class);
         $provider->updateTokens();
 
-        $this->assertEquals(0, count(CodeHelper::getCodesFromSession()));
+        $this->assertCount(0, CodeHelper::getCodesFromSession());
     }
 
-    public function testResultCreated()
+    public function testResultFound()
     {
-        $result = null;
+        /** @var Member $member */
         $member = $this->objFromFixture(Member::class, 'member1');
+        /** @var BootstrapMFAProvider $provider */
         $provider = Injector::inst()->get(BootstrapMFAProvider::class);
         $provider->setMember($member);
-        $provider->verifyToken('123345', $result);
+        $token = $member->encryptWithUserSettings('123345');
+        $result = $provider->fetchToken($token);
+
+        // No backupcodes generated yet
```

Comment:
You could make this an assertion failure message so it'd be a double whammy if the test fails:

```suggestion
        $this->assertNull($result, 'Backup codes should not have been generated yet');
```